src/page-modules/assistant/layout.tsx

```diff
@@ -24,13 +24,15 @@ export type AssistantLayoutProps = PropsWithChildren<{
   initialFromFeature?: GeocoderFeature;
   initialToFeature?: GeocoderFeature;
   initialTransportModesFilter?: TransportModeFilterOption[] | null;
+  initialDepartureMode?: DepartureDate;
```

Comment:
It might be confusing that his is named `DepartureMode` and the type is `DepartureDate` when we have a type that is named `DeperatureMode`. Could change it to `initialDepartureDate` instead?